frontend/components/card/search-result/CardSearchResultOrganization.vue

```diff
@@ -2,11 +2,11 @@
   <div class="flex flex-col w-full md:flex-row">
     <div class="flex justify-center w-full md:w-fit">
       <div
-        class="border rounded-lg w-fit border-light-section-div dark:border-dark-section-div bg-light-content"
+        class="border rounded-lg w-fit border-light-section-div dark:border-dark-section-div bg-light-content dark:bg-dark-content"
```

Comment:
This isn't needed as we want it to be white for now so the back tech from below logo shows up, but also no stress 😊